package.json

```diff
@@ -1,9 +1,9 @@
 {
   "private": true,
   "scripts": {
-    "heroku-postbuild": "NODE_OPTIONS=--max_old_space_size=4096 webpack --config webpack/webpack.config.prod.js --progress && tsc --project tsconfig.consumer.json",
+    "heroku-postbuild": "NODE_OPTIONS=--max_old_space_size=4096 webpack --config webpack/webpack.config.prod.js --progress && tsc --project server/tsconfig.json",
```

Comment:
So -- the server's tsconfig.json is sufficient for the consumer?

Reply:
Same file just renamed it to regular `tsconfig.json` and moved it into `/server` because the file is only used for scripts/files in that directory.